### DISCUSSION TITLE
[UI] Multiple archive selection in archives tab would make diffing more efficient

### DISCUSSION THREAD

**Discussion by jose1711** [borgbase/vorta#988] [Ideas]
If you want to see the diff between two archives you need to click Diff, then the new dialog opens (same as the current view) and you select two archives there. I think it would be better if the Archives view allows one to select two archives directly and diff displays the differences instead of an additional step.

**Comment by m3nu** [comment]
IIRC, we tried this, but there was some issue. But if you know how to solve this, just open a small PR and we'll merge it.
